### DISCUSSION TITLE
Display Channel name in multi-tenant solution

### DISCUSSION THREAD

**Discussion by blochaou** [vendurehq/vendure#1934] [Q&A]
Hi
I would like to display the channel name in front-store for each product. Unfortunately the graphQL product query in shop-api does not return the channel variable. Please what is the best way to return the channel relative to the product ?
Regards

**Comment by michaelbromley** [comment]
Hi,

To do this, you can define a new field on the Product type and then create a field resolver that looks up the product's channels and returns the data you need. You can find docs on that here: https://www.vendure.io/docs/plugins/plugin-examples/extending-graphql-api/#add-fields-to-existing-types
